fhevm/tee_interpreter.go

```diff
@@ -405,57 +405,33 @@ func marshalTfheType(value any, typ tfhe.FheUintType) ([]byte, error) {
 func unmarshalTfheType(data []byte, typ tfhe.FheUintType) (uint64, error) {
```

Comment:
@lovenoble This whole function could be removed, and replaced by one line:
```go
result := big.NewInt(0).SetBytes(data).Uint64()
```

That was the reason I wrote this comment:

https://github.com/Inco-fhevm/fhevm-go/blob/f0c6ba75e84bce4887145bd5a9fb9aa2728cb59f/fhevm/tee_interpreter.go#L45-L48